.github/workflows/new-release-branch.yaml

```diff
@@ -1,8 +1,40 @@
 name: New Release Branch
 
 on:
-  repository_dispatch:
-    types: [new-release-branch]
+  workflow_dispatch:
+    inputs:
+      branch_name:
+        description: Release branch name to create
+        required: true
+        type: number
```

Comment:
Shouldn't this be `string`?

Reply:
You're totally right, copy paste error, blah